docs/enhancement-proposals/off-ep-0002.md

```diff
@@ -0,0 +1,85 @@
+# OFF-EP 0002 — Support vdW interactions with a hard cut-off
+
+**Status:** Proposed
+
+**Authors:** Matt Thompson
+
+**Stakeholders:** David Mobley, John Chodera, Jeffrey Wagner, Simon Boothroyd
+
+**Acceptance criteria:** <Unanimity>
+
+**Created:** 2021-11-23
+
+**Discussion:** [Issue #7](https://github.com/openforcefield/standards/issues/7)
+
+**Implementation:** [PR #22](https://github.com/openforcefield/standards/pull/22)
+
+## Abstract
+
+This OFF-EP adds a `none` option in the `long_range_dispersion` option in the vdW section of the
+SMIRNOFF specification.
+
+## Motivation and Scope
+
+In most use cases involving force fields, vdW interactions are truncated at cutoff distance and have
+no effect past that cutoff. There [are ways](https://pubs.acs.org/doi/10.1021/jp980939v) to
+approximately correct for this, known as long-range dispersion corrections, that can improve the
+accuracy and/or speed of simulations. The SMIRNOFF specification currently has an option that
+encodes this: `long_range_dispersion`, with a default value of `"isotropic"`. There are no other
+supported values and no way to bypass adding this correction. While potentially dubious for use in
```

Comment:
"While potentially dubious for use scientific applications, users may wish to not include this correction."

This is not a compelling motivation for this addition.

Can you provide a compelling motivation (e.g. a clear and specific use case)?

Reply:
I think the motivation partly is something like, "We would like to validate that we have correctly implemented the force field aside from our treatment of long-range LJ interactions," as implementations of these add substantial additional complexities.

Reply:
The motivation should be made explicit and compelling in this OFF-EP before it is accepted.

Reply:
@davidlmobley described accurately what I'm after here, and I updated the text in an effort to reflect that

Reply:
Did you push your changes? I'm still seeing "While potentially dubious for use in scientific applications, users may wish to not include this correction."

Reply:
See https://github.com/openforcefield/standards/blob/f585e32672c115f84197ecdf306d0505a66bb60a/docs/enhancement-proposals/off-ep-0002.md?plain=1#L29-L33